policyengine_us/tests/policy/baseline/gov/states/me/tax/income/deductions/me_deduction.yaml

```diff
@@ -0,0 +1,19 @@
+- name: ME deduction given Single status, income means deduction won't be limited
```

Comment:
- name: ME deduction given Single status, income of $80k means deduction won't be limited